### PR TITLE
Refactor/upgrading to node 20 11 0

### PR DESCRIPTION
**Pull request by giulianok**
## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update
- [ ] Security update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

### CODE REVIEW

**Review by viveks-dev — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ARG VERSION=lts
+ARG VERSION=20.11.0
```
Comment (PixnBits) [inline Dockerfile:1]: Revert, lts is the default value for convenience but all workflows cat .nvmrc for repeatability

File: package-lock.json [hunk 2]
```diff
@@ -473,25 +473,25 @@
       }
     },
     "node_modules/@babel/compat-data": {
-      "version": "7.23.5",
-      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.23.5.tgz",
```
Comment (PixnBits) [inline package-lock.json:477]: These changes seem unrelated, are the packages' existing versions incompatible with Node.js 20.x?
